addons/makeslave.pl

```diff
@@ -2,36 +2,56 @@
 
 use Config::IniFiles;
 
+BEGIN {
+    # log4perl init
+    use constant INSTALL_DIR => '/usr/local/pf';
+    use lib INSTALL_DIR . "/lib";
+}
+
+use pf::file_paths qw($pf_config_file);
+
 my %ini;
 
 tie %ini, 'Config::IniFiles';
 
-my $pfconf = "/usr/local/pf/conf/pf.conf";
-
 my %inipfconf;
-tie %inipfconf, 'Config::IniFiles', ( -file => "$pfconf" );
+tie %inipfconf, 'Config::IniFiles', ( -file => "$pf_config_file" );
 
 my $replication_username = $inipfconf{"active_active"}{"galera_replication_username"};
 my $replication_password = $inipfconf{"active_active"}{"galera_replication_password"};
 
+if ($replication_username eq "" | $replication_password eq "") {
+    die "replication information are missing";
```

Comment:
*is missing

Also would be good to say its active_active.galera_replication_username and active_active.galera_replication_password